en/03_Drawing_a_triangle/02_Graphics_pipeline_basics/01_Shader_modules.adoc

```diff
@@ -205,8 +205,8 @@ Create a `compile.bat` file with the following contents:
 
 [,bash]
 ----
-C:/VulkanSDK/x.x.x.x/Bin/glslc.exe shader.vert -o vert.spv
-C:/VulkanSDK/x.x.x.x/Bin/glslc.exe shader.frag -o frag.spv
+%VULKAN_SDK%/Bin/glslc.exe shader.vert -o vert.spv
+%VULKAN_SDK%/Bin/glslc.exe shader.frag -o frag.spv
```

Comment:
$VULKAN_SDK for bash

Reply:
Thanks :)

Reply:
Oddly that env variable isn't set on m Ubuntu setup (WSL2), even though I've installed the SDK :/